app/controllers/admin/schedules_controller.rb

```diff
@@ -28,6 +28,25 @@ def index
       end
     end
 
+    def create
+      invoke_callbacks(:create, :before)
+      @object.attributes = permitted_resource_params
+      @object.save!
+
+      @object.order_cycle_ids = params[:order_cycle_ids]
+      if @object.save
+        invoke_callbacks(:create, :after)
+        flash[:success] = flash_message_for(@object, :successfully_created)
+        respond_with(@object) do |format|
+          format.html { redirect_to location_after_save }
+          format.js   { render layout: false }
```

Comment:
there's only one way to create a schedule in the OC page. I think you should delete the format.html here.
This is the new schedule request accept:
![image](https://user-images.githubusercontent.com/1640378/83183058-2a107080-a11f-11ea-8828-7e4b25402220.png)